src/blog/2023/07/images-in-node-red-dashboards.md

```diff
@@ -24,6 +24,66 @@ When designing a dashboard, images allow you to significantly enrich your conten
 
 - displaying physical tools which should be used to resolve a problem.
 
+## Prerequisites
+
+Before proceeding, ensure that you have the following custom nodes installed, as they will be used throughout this guide:
+
+- [node-red-contrib-calc](https://flows.nodered.org/node/node-red-contrib-calc) - A Node-RED node to perform basic mathematical calculations.
+- [node-red-contrib-image-output](https://flows.nodered.org/node/node-red-contrib-image-output) - A simple way to preview and examine images in your flows.
+- [node-red-contrib-os](https://flows.nodered.org/node/node-red-contrib-os) - Nodes for obtaining system information like CPU usage.
+- [node-red-contrib-string](https://flows.nodered.org/node/node-red-contrib-string) - A string manipulation node based on the lightweight stringjs library.
+- [@flowfuse/node-red-dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) - A set of dashboard nodes for Node-RED.
+- [node-red-node-base64](https://flows.nodered.org/node/node-red-node-base64) - A Node-RED node to encode and decode data to and from base64.
+
+## Easily Add Images to Node-RED Dashboards with FlowFuse’s Static Asset Service
+
+[FlowFuse's static assets](https://flowforge.com/docs/user/static-asset-service/) service provides a convenient way to manage static assets, such as images, within Node-RED instances. This service streamlines asset management by allowing everything to be handled through the same interface. If you want to display images using this service, follow the steps below for a straightforward process.
+
```

Comment:
@gayanSandamal Can you improve on these instructions? It seems like it's a lot of work to use images in the dashboard?

Reply:
I'm looking into this

Reply:
@ZJvandeWeg Simplified instructions by grouping and reducing steps for clarity with the following [commit](https://github.com/FlowFuse/website/pull/2612/commits/2dc4502cb6a62d10d0c7bd362071cb68f5a45f80) 

### Previously
**There were 11 bullet points**
<img width="253" alt="Screenshot 2024-09-26 at 18 46 11" src="https://github.com/user-attachments/assets/5046bbce-7af9-46f4-8df4-05e8eacb5174">

### After
**There are only 8 bullet points by grouping simplifying all the 11 bullet points**
<img width="495" alt="Screenshot 2024-09-26 at 19 17 45" src="https://github.com/user-attachments/assets/409405ac-acc1-454e-b4b8-e728e179bd3f">
<img width="510" alt="Screenshot 2024-09-26 at 19 18 09" src="https://github.com/user-attachments/assets/cca454ac-63d2-445a-87fb-d0b5c45e8d09">